### PR TITLE
Intern Task Alina Mulik: Homework

### PR DESCRIPTION
**Pull request by alina-mulik**
Added file with the solutions for Homework 3 tasks.

### CODE REVIEW

File: AlinaMulikSaysHello/Homework3.cs [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+            Console.WriteLine($"{number1.GetType()}");
+            byte number2 = 4;
+            Console.WriteLine($"{number2.GetType()}");
+            string hello3 = "Hello";
+            Console.WriteLine($"{hello3.GetType()}");
+            char r4 = 'R';
+            Console.WriteLine($"{r4.GetType()}");
+            double number5 = 23.093433222;
+            Console.WriteLine($"{number5.GetType()}");
+            int number6 = 40000;
```
Comment (yauhenists) [inline AlinaMulikSaysHello/Homework3.cs:100]: ushort is also possible

File: AlinaMulikSaysHello/Homework3.cs [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            Console.WriteLine("Task 3");
+            Console.WriteLine("Enter the number:");
+            var n = Console.ReadLine();
+            try
+            {
+                var parsedN = Int32.Parse(n);
+                var result = parsedN + parsedN * 2;
+                Console.WriteLine(result);
+            }
+
```
Comment (yauhenists) [inline AlinaMulikSaysHello/Homework3.cs]: Not a mistake - it's not needed empty line before 'catch' block. Below as well

File: AlinaMulikSaysHello/Homework3.cs [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+        private static void Task7()
+        {
+            Console.WriteLine("Task 7");
+            Console.WriteLine("Enter a number:");
+            var num = Console.ReadLine();
+            try
+            {
+                var convertedNum = Convert.ToInt32(num);
+                Console.WriteLine(
+                    (convertedNum % 2 == 0) ? "The entered number is even." : "The entered number is odd.");
```
Comment (yauhenists) [inline AlinaMulikSaysHello/Homework3.cs:184]: Not a mistake - not required brackets for the first operand

**Review by yauhenists — APPROVED** [review]

File: AlinaMulikSaysHello/Homework3.cs (hunk 1 above)
Comment (alina-mulik) [reply AlinaMulikSaysHello/Homework3.cs:100]: got it, thanks

File: AlinaMulikSaysHello/Homework3.cs (hunk 2 above)
Comment (alina-mulik) [reply AlinaMulikSaysHello/Homework3.cs]: will keep this in mind

File: AlinaMulikSaysHello/Homework3.cs (hunk 3 above)
Comment (alina-mulik) [reply AlinaMulikSaysHello/Homework3.cs:184]: got it, thank you

**Pull request merged**